stdlib/sys/process.md

```diff
@@ -53,6 +53,37 @@ enum Signal {
 
 Signals that can be sent to the host system.
 
+Variants:
+
+|name|data|description|
+|----|----|-----------|
+|`HUP`||Hangup.|
+|`INT`||Terminate interrupt signal.|
+|`QUIT`||Terminal quit signal.|
+|`ILL`||Illegal instruction.|
+|`TRAP`||Trace/breakpoint trap.|
+|`ABRT`||Process abort signal.|
+|`BUS`||Access to an undefined portion of a memory object.|
+|`FPE`||Erroneous arithmetic operation.|
+|`KILL`||Kill.|
+|`USR1`||User-defined signal 1.|
+|`SEGV`||Invalid memory reference.|
+|`USR2`||User-defined signal 2.|
+|`PIPE`||Write on a pipe with no one to read it.|
+|`ALRM`||Alarm clock.|
+|`TERM`||Termination signal.|
+|`CHLD`||Child process terminated, stopped, or continued.|
+|`CONT`||Continue executing, if stopped.|
+|`STOP`||Stop executing.|
+|`TSTP`||Terminal stop signal.|
+|`TTIN`||Background process attempting read.|
+|`TTOU`||Background process attempting write.|
+|`URG`||High bandwidth data is available at a socket.|
+|`XCPU`||CPU time limit exceeded.|
+|`XFSZ`||File size limit exceeded.|
+|`VTALRM`||Virtual timer expired.|
```

Comment:
It seems that the enum variants without a description aren't included

Reply:
I did this intentionally. Since the full enum definition is already printed I didn't see much of a point to having an entry with just the variant name and no context

Reply:
That's fine when we remove the table for variants. If it were a table like you did originally, it should be the full enumeration.

Reply:
This would also apply to record fields though. Unless we want to ditch the table for those as well?

Reply:
Looking at my layout suggestion, record fields remain in a table, which contains all fields.